src/views/TeamPage/components/Subsystems/Subsystems.js

```diff
@@ -1,83 +1,75 @@
-                        stack of Anahita consists of dedicated layers for the hardware integration, 
-                        controls and navigation, motion planning, perception and acoustic localization. 
-                        The software subsytem carries out simulations Gazebo and UWSim to ensure  that 
-                        things work smoothly before actual testing.The main aim of software subsystem is 
-                        to write code, that is generic and error free, in order efficiently coordinate the
-                         vehicles motion and enable it perform at full potential. </p>
-                    </Col>
-                    <Col md="2"></Col>
-                    
-                    <Col md="5" className="text-center">
-                        <h3 className="mt-0 mb-3 small-heading">Electrical</h3>
-                        <img src={electrical} className="  w-100"></img>
-                        <p className="auv-description-primary">The Electrical Subsystem builds the electrical framework 
-                        powering our AUV. We work on the electrical aspects of the AUV, providing
-                        power, driving actuators and interfacing with various sensors on board. We 
-                        build custom made PCBs to meet the specific requirements of the tasks. The
-                         Electrical framework consists of Power Distribution System and the Monitoring 
-                         system. Arduino Mega is used to control the actuation and read
-                        data from various sensors which in turn is controlled by an Intel NUC. 
-                        The main aim of the Electrical subsystem is to manage electrical setup and 
-                        act as an interface between the software and mechanical subsystems. </p>
-                    </Col>
-                    <Col md="5" className="text-center">
-                        <h3 className="mt-5 mb-3 small-heading mb-2">Mechanical</h3>
-                        <img src={mechanical} className="  w-100"></img>
-                        <p className="auv-description-primary">The mechanical subsystem if responsible for design and
-                        manufacture of the vehicle. We design the entire vehicle and accessories - 
-                        working on Fluid Dynamics, Actuator controls, and Pneumatics. We plan, simulate 
-                        and test the physical structure of the AUV and then manufacture it
-                        using state of art manufacturing techniques. We use latest software such as SolidWorks and ANSYS 
-                        to make the design robust, modular, light and strong. </p>
-                    </Col>
-                    <Col md="2"></Col>
-                    <Col md="5" className="text-center">
-                        <h3 className="mt-5 mb-3 small-heading mb-2">Business</h3>
-                        <img src={business} className="  w-100"></img>
```

Comment:
We would like to have some graphic images for each subsystem section. Do not remove any images.

Reply:
@ritam2358 once you have addressed a particular issue on which the reviewer has commented, reply with a comment or resolve the issue (if you are sure about your solution).

Reply:
I resolved the issue commented as well